changelog/2023-03-01T10_02_55+01_00_generalized_periodToHz

```diff
@@ -0,0 +1 @@
+Changed: Generalized `periodToHz`. Use type applications (`periodToHz @Natural @(Ratio Natural)`) to get the old behaviour back, in case type errors arise.
```

Comment:
```suggestion
CHANGED: Generalized `periodToHz`. Use type applications (`periodToHz @Natural @(Ratio Natural)`) to get the old behavior back, in case type errors arise.
```